Client/Frontend/Home/HomepageContextMenuHelper.swift

```diff
@@ -25,13 +25,18 @@ class HomepageContextMenuHelper: HomepageContextMenuProtocol {
     typealias ContextHelperDelegate = HomepageContextMenuHelperDelegate & UIPopoverPresentationControllerDelegate
     typealias SendToDeviceDelegate = InstructionsViewDelegate & DevicePickerViewControllerDelegate
     private var viewModel: HomepageViewModel
+    private let toastContainer: UIView
     weak var sendToDeviceDelegate: SendToDeviceDelegate?
-
+    weak var browserNavigationHandler: BrowserNavigationHandler?
     weak var delegate: ContextHelperDelegate?
     var getPopoverSourceRect: ((UIView?) -> CGRect)?
 
-    init(viewModel: HomepageViewModel) {
+    init(
+        viewModel: HomepageViewModel,
+        toastContainer: UIView? = nil
```

Comment:
Same, do we need this default param to `nil` and optional if we always have a `toastContainer` ? (same comment applies to homepage view controller). I know this will imply making changes in tests cases, but we can always create a helper function to create the subject to test if there's a lot of params to set in there.

Reply:
No indeed it's always not nil. i'm going to modify it